tests/functional/pv/pv_services/test_pvc_assign_pod_node.py

```diff
@@ -49,7 +48,6 @@ def verify_access_token_notin_odf_pod_logs(self):
             error_msg in pod_log
         ), f"Logs should not contain the error message '{error_msg}'"
 
-    @acceptance
```

Comment:
Should we mention in the comment that this test is also covered in acceptance test suite but in different approach and link to the test file?
Also would mention that if someone will modify this test case, to don't forgot to update the logic also in the acceptance suite test if needed.

Reply:
Good point, added comments for each testcase.